ddl/db_test.go

```diff
@@ -2284,6 +2284,32 @@ func (s *testDBSuite1) TestCreateTable(c *C) {
 	//tk.MustGetErrCode(failSQL, errno.ErrDuplicatedValueInType)
 	//_, err = tk.Exec("create table t_enum (a enum('B','b'));")
 	//c.Assert(err.Error(), Equals, "[types:1291]Column 'a' has duplicated value 'B' in ENUM")
+
+	// test for table option "union" not supported
+	tk.MustExec("use test")
+	tk.MustExec("CREATE TABLE x (a INT) ENGINE = MyISAM;")
+	tk.MustExec("CREATE TABLE y (a INT) ENGINE = MyISAM;")
+	tk.MustExec("INSERT INTO x VALUES (1);")
+	tk.MustExec("INSERT INTO y VALUES (2);")
```

Comment:
Could the above two lines be removed? It seems unnecessary.

Reply:
@zimulala 
sorry for late response
fixed in https://github.com/pingcap/tidb/pull/17995/commits/ddb75be7718b62f09829a189de688ce533bc712b
PTAL, thanks!